src/app/controllers/blogcontroller.js

```diff
@@ -70,6 +70,27 @@ class BlogController {
           GenericRes.error(res, 404, error.message);
         } 
       }
+
+      static async updateBlog(req, res, next) { 
+        try { 
+          const userId = await ExtractToken.authentication(req, res, next);
+          const blogId = req.params.post_id;
+          const { title, description } = req.body;
+          const blogModel = { title, description };
+          
+          const data = await BlogService.updateBlog(blogModel, blogId);
+          
+         if (!data) { 
+           GenericRes.error(res, 400, 'no blog', next);
+         }
+         const data1 = await BlogService.getOneBlog(blogId);
```

Comment:
I can suggest you to use the data store in the `data` variable after the if statement on line 85 to avoid multiple request on the database.

Reply:
Thanks... Let me  work on it!